cmd/start.go

```diff
@@ -217,6 +221,7 @@ func mountsFromFlag(mounts []string) []config.Mount {
 }
 
 func setDefaults(cmd *cobra.Command) {
+	startCmdArgs.VMType = "qemu"
```

Comment:
This was the key to getting it to work on macOS 12. But as mentioned elsewhere, we may have trouble with existing configuration files that have the assumption of default values being a particular thing, as the validation insists on an actual value.